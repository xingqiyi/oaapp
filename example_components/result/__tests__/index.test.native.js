import React from 'react';
import renderer from 'react-test-renderer';
// import { shallow } from 'enzyme';

import Result from '../index';

describe('Result', () => {
  it('renders correctly', () => {
    const wrapper = renderer.create(
      <Result
        imgUrl={{ uri: 'https://zos.alipayobjects.com/rmsportal/qlMpMyJTcSjMpKAgtcEt.png' }}
        title="验证成功"
        message="所提交内容已成功完成验证"
      />,
    );
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
