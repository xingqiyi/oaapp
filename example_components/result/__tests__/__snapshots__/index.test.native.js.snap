// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Result renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#ffffff",
        "borderBottomColor": "#dddddd",
        "paddingVertical": 21,
      },
      undefined,
    ]
  }
>
  <View
    style={
      Object {
        "margin": 0,
      }
    }
  >
    <Image
      source={
        Object {
          "uri": "https://zos.alipayobjects.com/rmsportal/qlMpMyJTcSjMpKAgtcEt.png",
        }
      }
      style={
        Object {
          "height": 60,
          "width": 60,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "marginTop": 15,
        "paddingHorizontal": 15,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#000000",
          "fontSize": 21,
        }
      }
    >
      验证成功
    </Text>
  </View>
  <View
    style={
      Object {
        "marginTop": 15,
        "paddingHorizontal": 15,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#888888",
          "fontSize": 16,
        }
      }
    >
      所提交内容已成功完成验证
    </Text>
  </View>
</View>
`;
